app/lti.go

```diff
@@ -4,14 +4,48 @@
 package app
 
 import (
+	"encoding/json"
+	"fmt"
 	"net/http"
 
+	"github.com/pkg/errors"
```

Comment:
@hrishavkmr can you change this package to "errors"?

I've had to update the `GetUserByLTI` function to use the "errors" package, and I don't know how to use both. This will need to be figured out before I can merge. This package is going into maintenance mode because of Go2 error proposals (see https://github.com/pkg/errors#roadmap)

See https://blog.dharnitski.com/2019/09/09/go-errors-are-not-pkg-errors/ for the differences.

Reply:
@mlippert I have done the required changes in PR #35.